pro/src/screens/CollectiveOfferSelectionDuplication/CollectiveOfferSelectionDuplication.module.scss

```diff
@@ -0,0 +1,67 @@
+@use "styles/mixins/_rem.scss" as rem;
+@use "styles/variables/_colors.scss" as colors;
+
+.search-container {
+    position: relative;
+    margin: rem.torem(16px) 0;
+  }
+
+.search-button {
+    position: absolute;
+    right: rem.torem(4px);
+    top: rem.torem(4px);
+    height: rem.torem(32px);
+}
+
+.offer-info {
+  font-size: rem.torem(12px);
+  font-weight: 600;
+}
+
+.offer-selected {
+  background-color: colors.$light-lavender;
+}
+
+.offer-selection {
+  padding: rem.torem(16px);
+  border-bottom: rem.torem(1px) solid colors.$grey-medium;
+
+  &-label {
+    display: flex;
+    gap: rem.torem(16px);
+    align-items: center;
+  }
+
+  &:last-child {
+    border-bottom: initial;
+  }
+}
+
+.icon-button {
+    margin-right: rem.torem(8px);
+}
+
+.search-no-results {
+    color: colors.$grey-dark;
+    text-align: center;
+  
+    &-icon {
+      margin: rem.torem(10px) 0 rem.torem(30px);
+    }
+  
+    &-text {
+      margin-bottom: rem.torem(20px);
+    }
+  }
+
+.img-offer {
+  margin-left: rem.torem(16px);
+  width: rem.torem(80px);
+  height: rem.torem(80px);
+  border-radius: rem.torem(4px);
+}
+
+.offer-title {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
(detail) Pour info tu n'as pas de new line à la fin de ce fichier. Tu peux configurer ton IDE pour qu'il en mette une automatiquement au save.
Si tu veux le pourquoi, c'est pour [respecter le standard POSIX](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) :slightly_smiling_face: